OrbitGl/GraphTrack.cpp

```diff
@@ -32,15 +33,28 @@ void GraphTrack::Draw(GlCanvas* canvas, PickingMode picking_mode) {
   }
 
   float track_z = GlCanvas::kZValueTrack;
-  float text_z = GlCanvas::kZValueText;
 
   Box box(pos_, Vec2(size_[0], -size_[1]), track_z);
 
   // Draw label with graph value at current mouse position.
-  const Color kLabelTextColor(255, 255, 255, 255);
+  const Color kWhiteColor(255, 255, 255, 255);
+  const Color kBlackColor(0, 0, 0, 255);
   double graph_value = GetValueAtTime(time_graph_->GetCurrentMouseTimeNs());
-  canvas->GetTextRenderer().AddText(std::to_string(graph_value).c_str(), canvas->GetMouseX(), y0,
-                                    text_z, kLabelTextColor);
+
+  Vec2 white_text_box_size(layout.GetGraphTrackBoxWidth(), layout.GetTextBoxHeight());
```

Comment:
GetGraphTrackBoxWidth() is a constant, right? Isn't it possible that values are too long for the width chosen? I don't know if we have a good way of predicting text width based on a string, but it seems that's what would be needed here.

Reply:
Actually we may have a method to that that. Check TextRenderer::GetStringSize().